### ISSUE TITLE
When I generate the resource pack a player gets this

### ISSUE THREAD

**Issue by UnlikeDruid** [Ehhthan/HappyHud-Issues#96]
![2023-11-27_16 49 46](https://github.com/Ehhthan/HappyHud-Issues/assets/148410311/5938117b-acd2-450c-914d-e064b9ecf3b8)

**Comment by UnlikeDruid** [comment]
I should mention that it only happened to this player. I can see the huds without problems and many other players alike.

**Comment by MrRedCoffee** [comment]
Did you find any solution? I'm facing the same issue here.

**Comment by UnlikeDruid** [comment]
Yes, but there really is no specific solution. The player who predetermined
the problem changed his graphics card and with the new one it did not
happen again. That means it’s a client thing and not a server thing, at
least in my case. If you happen with more than one player and use plugins
like Oraxen or ItemsAdder, I recommend that you check the compatibility
configuration in the Happyhuds documentation. I hope I have been helpful
and do not hesitate to write if you have any questions
